docassemble/AssemblyLine/al_document.py

```diff
@@ -290,9 +308,7 @@ def value_if_defined(self) -> Any:
         Return the value of the field if it is defined, otherwise return an empty string.
         Addendum should never trigger docassemble's variable gathering.
         """
-        if defined(self.field_name):
-            return value(self.field_name)
-        return ""
+        return showifdef(self.field_name, "")
```

Comment:
@BryceStevenWilley please check my assumptions here--I think we had the wordier code to workaround an older, unnecessarily slow version of `showifdef()` but this simplification can be made now?

I'm looking at 

```python
def showifdef(var: str, alternative=''):
    """Returns the variable indicated by the variable name if it is
     defined, but otherwise returns empty text, or other alternative text.
     """
    # A combination of the preambles of defined and value
    str(var)
    if not isinstance(var, str):
        raise Exception("showifdef() must be given a string")
    if not re.search(r'[A-Za-z][A-Za-z0-9\_]*', var):
        raise Exception("showifdef() must be given a valid Python variable name")
    try:
        return eval(var, {})
    except:
        pass
    if re.search(r'[\(\)\n\r]|lambda:|lambda ', var):
        raise Exception("showifdef() is invalid: " + repr(var))
    return _defined_internal(var, DefCaller.SHOWIFDEF, alt=alternative)
```

and this seems to be a safe change but please double-check

Reply:
I'm not sure if we were working around anything, but this literally used to be the code of [showifdef](https://github.com/jhpyle/docassemble/blob/595ad0a08572c0188f9ab49b1a6af99a3621cd42/docassemble_base/docassemble/base/functions.py#L4443-L4445), so switching would make things slightly faster. Haven't come across any bugs in the new implementation yet, and it's bin in there for ~a year or so I think, so :+1: from me.